profile/src/main/java/org/openedx/profile/presentation/edit/EditProfileFragment.kt

```diff
@@ -627,28 +635,32 @@ private fun EditProfileScreen(
                                         .clip(CircleShape)
 
                                         .noRippleClickable {
-                                            isOpenChangeImageDialogState = true
-                                            if (!uiState.account.isOlderThanMinAge()) {
-                                                openWarningMessageDialog = true
+                                            if (!uiState.account.requiresParentalConsent) {
```

Comment:
we can define `uiState.account.requiresParentalConsent` as a local variable because of multiple usage.